src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -178,14 +204,25 @@ export default Relay.createContainer(ConversationSnippet, {
         }
         last_message
         last_message_at
-        artworks {
-          id
-          href
+        items {
           title
-          date
-          artist_names
-          image {
-            url
+          item {
+            __typename
+            ... on Artwork {
+              id
```

Comment:
I guess just habit with Relay, always include `id`!

You're right tho, we don't need it.